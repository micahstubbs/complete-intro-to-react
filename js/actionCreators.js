// @flow

import { SET_SEARCH_TERM } from './actions';

// prettier-ignore
export function setSearchTerm(searchTerm) { // eslint-disable-line import/prefer-default-export
  return { type: SET_SEARCH_TERM, payload: searchTerm };
}
