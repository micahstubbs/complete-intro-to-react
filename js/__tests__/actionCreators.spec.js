// @flow

import { setSearchTerm, addAPIData } from '../actionCreators';

test('setSearchTerm', () => {
  expect(setSearchTerm('New York')).toMatchSnapshot();
});
