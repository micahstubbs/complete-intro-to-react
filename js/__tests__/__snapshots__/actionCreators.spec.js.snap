// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`setSearchTerm 1`] = `
Object {
  "payload": "New York",
  "type": "SET_SEARCH_TERM",
}
`;
